lib/email_transactional/reader.rb

```diff
@@ -0,0 +1,51 @@
+class EmailTransactional::Reader
+  EMAIL_REGEX = /(.*)--inline.html/
+  PATH_TO_EMAILS = '/emails'
+
+  # this will need to change, when used as a gem the local dir will be different
+  def self.default
+    new('./build')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.